covid-nlp/main.ipynb

```diff
@@ -0,0 +1,3181 @@
+{
```

Comment:
<p>Its hard to notice visually where the rgxlog cells end, can you simply make them into different code cells?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>